pkg/util/net/ip/ip.go

```diff
@@ -82,3 +85,28 @@ func isIPv6Disabled(filename string) bool {
 
 	return data[0] == '1'
 }
+
+func ChangeCIDRsToIPs(ifs *api.InterfaceStatus) error {
+	// if IP address was the passed value we should do nothing
```

Comment:
I am not fun of comments. If possible, please express this in the func doctext (I am fine with just dropping as well).